features/user/managing_customers/browsing_customers.feature

```diff
@@ -0,0 +1,17 @@
+@managing_customers
+Feature: Browsing customers
+    In order to see all customers in the store
+    As an Administrator
+    I want to browse customers
+
+    Background:
+        Given the store has customer "f.baggins@example.com"
+        And the store has customer "mr.banana@example.com"
+        And the store has customer "l.skywalker@example.com"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Browsing currencies in store
+        When I want to see all customers in store
+        Then I should see 4 customers in the list
```

Comment:
In the background we have 4 customers. This is confusing. I know that administrator appears in the list, but we should think about making this feature more clear. Ideally, we would display store staff in a separate panel, but I'm not sure how to to handle this right now.